program/src/state.rs

```diff
@@ -2299,7 +2299,7 @@ pub fn load_market_state<'a>(
             from_bytes_mut(mid)
         });
 
-    state.check_flags()?;
+    state.check_flags(false)?;
```

Comment:
What does the false do here?

Reply:
Serum added ability for Market to be disabled. In old version, default behavior was to error if Market was disabled. So by setting `allow_disabled` to false I preserve the old behavior.